.gitignore

```diff
@@ -26,3 +26,6 @@ gitignore
 # System Files
 .DS_Store
 Thumbs.db
+
+docs/assets/*.js
+docs/assets/*.css
```

Comment:
these are deployed as the static website on github pages, but we can remove it from `dev branch`

Reply:
Alrighty